ui/packages/atlasmap-core/src/services/file-management.service.ts

```diff
@@ -296,7 +297,8 @@ export class FileManagementService {
             (doc.inspectionSource !== null &&
```

Comment:
What is this if-clause checking? do we have any case we want to silently ignore on purpose? It looks like it's better to put the document metadata anyway here, or show an error if it finds any invalid document.

Reply:
Doc.Properties and DOC.Constants are skipped - we could flip the logic and just looks for those.

Reply:
Ah OK makes sense, and yes would be better to explicitly exclude constants/props, which exactly describes what's going on.